internal/tasks/common.go

```diff
@@ -1,13 +1,31 @@
 package tasks
 
 import (
+	"fmt"
+
 	api "github.com/flightctl/flightctl/api/v1alpha1"
 	"github.com/flightctl/flightctl/internal/store/model"
 	"github.com/flightctl/flightctl/internal/util"
 )
 
 const ItemsPerPage = 1000
 
+type ErrUnknownConfigName struct {
```

Comment:
No need to reimpliment error
```suggestion
var ErrUnknownConfigName = errors.New("failed to find configuration item name")
```

Reply:
Done